pkgs/build-support/vm/default.nix

```diff
@@ -744,6 +744,7 @@ rec {
 
       perl -w ${deb/deb-closure.pl} \
         ./Packages ${urlPrefix} ${toString packages} > $out
+      nixfmt $out
```

Comment:
How did this pass review?  @Korny666 @markuskowa

Reply:
Can you explain a bit further what you are trying to point out?
I am a bit confused 🫠